README.md

```diff
@@ -116,6 +116,70 @@ $ remme account get-balance \
 }
 ```
 
+### Batch
+
+Get a list of batches — ``remme batch get-list``:
+
+| Arguments | Type   |  Required | Description                                         |
+| :-------: | :----: | :-------: | --------------------------------------------------- |
+| ids       | String |  No       | Identifiers to get a list of batches by.            |
+| start     | String |  No       | Parameter to list batches starting from.            |
+| limit     | Integer|  No       | Parameter to limit amount of batches.               |
+| head      | String |  No       | Block identifier to get a list of batches from.     |
+| reverse   | String |  No       | Parameter to reverse result.                        |
+| node-url  | String |  No       | Node URL to apply the command to.                   |
+
+```bash
+$ remme batch get-list \
+      --ids=[c2eeb94926d3432e41cb5ceed078f78466389e4fe685ec958021a1368f634c035072e434e5d0ff64820dd01fde6e5afc67eb5a9ae6c48d3983ff43abe98aef6b] \
+      --node-url=159.89.104.9
```

Comment:
Provide the most full example. As [here](https://github.com/Remmeauth/remme-core-cli/pull/18/files#diff-04c6e90faac2675aa89e2176d2eec7d8R137).

Reply:
Done.